foundation/foundation-mda/src/main/resources/templates/deployment/pipeline-invocation-service/v2/pipeline.invocation.service.values-dev.yaml.vm

```diff
@@ -2,14 +2,8 @@
 aissemble-pipeline-invocation-chart:
   aissemble-quarkus-chart:
     image:
-      dockerRepo: ""
+      imagePullPolicy: IfNotPresent
```

Comment:
I: This is never supposed to hit a project-specific image, so setting `dockerRepo` to blank doesn't really make sense.